dnn/variational_inference.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<p>combine all these code blocks into a single block for 'model train'</p><p>maybe separate out 'isoscape generation' from model train.</p><p> </p><p>so:</p><ol><li>parameters</li><li>data prep</li><li>model def</li><li>model train</li><li>save isoscape</li></ol><p> </p><p>5 clicks max if you dont need to change params.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>